keyboards/madjax_macropad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    KC_P00 = SAFE_RANGE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+     *             ┌───┐
+     *             │ - │
+     * ┌───┬───┬───┼───┤
+     * │ 7 │ 8 │ 9 │ + │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │ % │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ = │
+     * ├───┼───┼───┼───┤
+     * │ 0 │00 │ . │Ent│
+     * └───┴───┴───┴───┘
+     */
+    [0] = LAYOUT_madjax(
+                                   KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,   KC_PERC,
+        KC_P1,   KC_P2,   KC_P3,   KC_EQL,
+        KC_P0,   KC_P00,  KC_PDOT, KC_PENT
+    ),
+
+    /*
+     *             ┌───┐
+     *             │ - │
+     * ┌───┬───┬───┼───┤
+     * │Hom│ ↑ │PgU│ + │
+     * ├───┼───┼───┤───┤
+     * │ ← │   │ → │ % │
+     * ├───┼───┼───┤───┤
+     * │End│ ↓ │PgD│ = │
+     * ├───┼───┼───┤───┤
+     * │Ins│   │Del│Ent│
+     * └───┴───┴───┘───┘
+     */
+    [1] = LAYOUT_madjax(
```

Comment:
This layer is also not accessible.
```suggestion
    [1] = LAYOUT(
```

Reply:
Done.

Reply:
This suggestion has not been made.

Reply:
Should be now.

Reply:
> This layer is also not accessible.

Reply:
Removed it.